Source/ACE/Entity/Creature.cs

```diff
@@ -495,5 +497,162 @@ protected virtual void UpdateVitalInternal(CreatureVital vital, uint newVal)
                 new ActionChain(this, () => VitalTickInternal(vital)).EnqueueChain();
             }
         }
+
+        public override void SerializeIdentifyObjectResponse(BinaryWriter writer)
```

Comment:
I think this is generally right then.  

If breaking creature identification for non-creatures DebugObjects is a problem, we may consider not doing this refactor yet.  

I would argue that there is little point returning a proper identification for an improperly implemented underlying type, but I'm not working with general WorldObject properties or their uses, and I have a limited view in this realm.